docs/modules/ROOT/pages/tutorial/package.adoc

```diff
@@ -85,6 +85,13 @@ This gives us access to the provided classes.
 
 This will add the component cert-manager to the cluster and configure the lestsencrypt mail to `cert@example.com`.
 
+[NOTE]
+====
+During catalog compilation, you might see several warnings like `[WARNING] Reclass class not found: 'cert-manager.common'. Skipped!`.
+This is expected, as Commodore first has to render the inventory to even be able to clone packages.
+Still, double-check the manifests to ensure the parameters really get applied.
```

Comment:
```suggestion
```

If you include a class which isn't found after making packages available, the compilation will fail, so if compilation succeeds with the package included, parameters defined in the package class are always applied.